src/components/PurchaseOrder/POForm.js

```diff
@@ -340,4 +340,5 @@ export default stripesForm({
   enableReinitialize: true,
   form: PO_FORM_NAME,
   navigationCheck: true,
+  destroyOnUnmount: true,
```

Comment:
it would be nice to hear the purpose of this change, since it has a quite big impact

Reply:
I just forgot to delete this line